pipeline-scripts/buildlib.groovy

```diff
@@ -1174,6 +1174,19 @@ def getGroupBranch(doozerOpts) {
     return branch
 }
 
+def cleanWorkspace() {
+    cleanWs(cleanWhenFailure: false, notFailBuild: true)
+    dir("${workspace}@tmp") {
```

Comment:
Sometimes the dir is named as `@<number>` or `@<number>@tmp`, would like to use a wildcard to delete them as well.

Reply:
Yeah, so as I understand after reading [here](https://issues.jenkins.io/browse/JENKINS-30231), these are additional workspaces created when concurrent builds are running on the same node. In that case I would assume the workspace dirs (like job@2) get deleted with cleanWs statement, and other dirs like job@2@tmp get deleted with subsequent statements. But that is what I'm assuming, it would be nice if I can actually test that..

Reply:
hey @vfreex do you think we can merge this? And then we can see if the issue persist?